app/views/service-patterns/parking-permit/example-service/resident-choice.html

```diff
@@ -7,10 +7,24 @@
 
 <div class="column-two-thirds">
 
-
+		{% if council.string == 'oxfordshire' %}
     <p>
-      You need a permit to park here because {{postcode}} is in the <a href="parking-zone-map">{{council.parkingBoundary}} parking zone</a>.</p>
-    <form class="radio-nav">
+      You need a permit to park here because {{postcode}} is in the <a href="parking-zone-map">East Oxford parking zone</a>.
```

Comment:
Can you change the text here back to `{{council.parkingBoundary}}` and update your `parkingBoundary` value in `councils.json`

Reply:
As I understand "East Oxford" should uppers only on this page (@catherinewoodoxfordshire is this is correct?) and If I will update parkingBoundary  value in  councils.json it will change it globally